runtime/Cpp/runtime/src/atn/SetTransition.cpp

```diff
@@ -11,8 +11,8 @@
 using namespace antlr4;
 using namespace antlr4::atn;
 
-SetTransition::SetTransition(TransitionType transitionType, ATNState *target, misc::IntervalSet aSet)
-  : Transition(transitionType, target), set(aSet.isEmpty() ? misc::IntervalSet::of(Token::INVALID_TYPE) : std::move(aSet)) {
+SetTransition::SetTransition(ATNState *target, misc::IntervalSet aSet)
```

Comment:
The `NotSetTransition` constructor still calls the constructor  with the `TransitionType` parameter which is now gone.

Reply:
That was an unrelated change that I thought I undid, apparently I missed that bit. Fixed.